src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz.erl

```diff
@@ -328,7 +329,9 @@ is_authorized_on_resource(RequestorId, ResourceType, ResourceId, ActorType, Acto
         %% This api returns not found for any missing permission
         {error, not_found} -> false;
         %% Otherwise, we expect server_error; not forbidden
-        {error, server_error} -> {error, server_error}
+        {error, server_error} -> {error, server_error};
```

Comment:
Let's change this to have any other error tuple returned as-is, eg `{error, Any} -> {error, Any}`; the shape of the tuple is more important to capture than each individual possible error type.